lib/mapper.js

```diff
@@ -22,13 +52,52 @@ exports.track = function(track){
     objCase.del(props, 'revenue');
   }
 
-  return {
-    email: track.email(),
-    userId: track.userId(),
-    eventName: track.event(),
-    createdAt: unixTime(track.timestamp()),
-    dataFields: props
-  };
+  var eventName = track.event();
+  var cart = props["products"] || props["cart"];
```

Comment:
The ecommerce examples [here](https://segment.com/docs/spec/ecommerce/#completing-an-order) are examples of spec-compliant calls. So to take advantage of ecommerce specific integrations in downstream integrations they have to adhere to that standard (or else maintaining each separately would get too crazy). In other words, we wouldn't support the mapping for users who pass `cart` in lieu of `products` — we'd require they send `products` instead. I know it sounds restrictive, but it makes everything easier to have some mediation/schema/spec dictating the shape of the data :)